.gitignore

```diff
@@ -147,3 +147,5 @@ cython_debug/
 docker-compose.dev.yaml
 karton-logs
 output
+Artemis-modules-extra
+/artemis/help_sql_error.py
```

Comment:
I don't think we need to gitignore that file

Reply:
This is for my development version. I will remove it before pushing the changes.